### ISSUE TITLE
Make movies of the pngs

### ISSUE THREAD

**Issue by acviana** [STScI-Citizen-Science/MTPipeline#57]
I want to create movies showing all the images in a folder. I can do this using `ffmpeg`, here is an example that works for all the cr rejected files in a folder.

```
ffmpeg -f image2 -r 1 -pattern_type glob -i '*cr*wide_single_sci_linear.png' video.mp4
```

**Comment by acviana** [comment]
The files are now in central storage. Once you have access to this area start using `ffmpeg` to create movies. I would like a folder in `drizzled/` called `movies`. There should be 6 movies for each `<proposal id>_<object_name>/png/` folder (e.g. `12345_planetx/png`):
1. All the wide images
2. The cosmic ray rejected wide images
3. The non-cosmic ray rejected wide images 
4. All the center images
5. The cosmic ray rejected center images
6. The non-cosmic ray rejected center images 

So there should be (6 \* number of folders) images in that folder. It would also be good if you could figure out how to get the images to switch roughly twice per second.

I suggest doing all this in python using the `subprocess.call` function and adding the file to the `tools` folder.

**Comment by prathamjoshi** [comment]
I now have `ffmpeg` working and have sent an e-mail to IT (CCd to you) about getting access to the central storage. Is there a sample of images I could use to work on the script while I wait on getting access?

**Comment by prathamjoshi** [comment]
It seems I do not have write-access to create folders or files under `drizzled` directory.

**Comment by prathamjoshi** [comment]
So i have pushed the script to my forked `MTpipeline` branch: [MTpipeline fork](https://github.com/prathamjoshi/MTPipeline/blob/master/tools/mpegCreator.py) 
Having issues with `ffmpeg` and image/vidoe resolution. am going to look more into how ffmpeg works with pngs

**Comment by acviana** [comment]
You can close this.

**Comment by prathamjoshi** [comment]
ok closes #57

**Comment by prathamjoshi** [comment]
ohh wait does not look like I can since it is in the StSci branch.
